app/models/idea.rb

```diff
@@ -1,3 +1,5 @@
 class Idea < ActiveRecord::Base
+  validates :title, format: { with: /\A[a-zA-Z0-9,?! ]+\z/ }
+  validates :body, format: { with: /\A[a-zA-Z0-9,?! ]+\z/ }
```

Comment:
@rrgayhart interesting... could this look like a switch statement with the regex filtering out bad characters within the create function?